package.json

```diff
@@ -8,13 +8,9 @@
         "url": "https://github.com/kubernetes-client/javascript.git"
     },
     "files": [
-        "dist/*.ts",
```

Comment:
@brendandburns I guess we could use these patterns as a mechanism to only include certain source maps (referring to your comment here https://github.com/kubernetes-client/javascript/issues/2249#issuecomment-2676442930), but as I said on the other issue, I'm not sure that's worth doing.